src/permutat.c

```diff
@@ -1178,19 +1178,21 @@ Obj             PowPerm2Int (
 {
     Obj                 pow;            /* handle of the power (result)    */
     UInt2 *             ptP;            /* pointer to the power            */
-    UInt2 *             ptL;            /* pointer to the permutation      */
+    const UInt2 *             ptL;            /* pointer to the permutation      */
```

Comment:
Adjust indentation to match surrounding code? I.e. remove spaces before `ptL` to re-align the columns?